pkg/plugin/driver/daemonset/daemonset.go

```diff
@@ -78,19 +85,46 @@ func (p *Plugin) GetResultType() string {
 	return p.Definition.ResultType
 }
 
+//FillTemplate populates the internal Job YAML template with the values for this particular job.
+func (p *Plugin) FillTemplate(hostname string) (*bytes.Buffer, error) {
+	var b bytes.Buffer
+	// TODO (EKF): Should be YAML once we figure that out
+	container, err := utils.ContainerToJSON(&p.Definition.Spec)
```

Comment:
this could replace this with `containerBytes, err := json.Marshal(&p.Definition.Spec)` and then used as bytes all the way through or turned into a string if needed.

Same applies to job driver.

Reply:
This is all superceded by a YAML output, that'll be in a subsequent PR